magicblock-committor-service/src/persist/db.rs

```diff
@@ -385,114 +388,77 @@ impl CommittorDb {
         extract_committor_rows(&mut rows)
     }
 
-    pub(crate) fn get_commit_statuses_by_reqid(
+    pub(crate) fn get_commit_statuses_by_id(
         &self,
-        reqid: &str,
+        message_id: u64,
     ) -> CommitPersistResult<Vec<CommitStatusRow>> {
         let query =
-            format!("{SELECT_ALL_COMMIT_STATUS_COLUMNS} WHERE reqid = ?1");
+            format!("{SELECT_ALL_COMMIT_STATUS_COLUMNS} WHERE message_id = ?1");
         let stmt = &mut self.conn.prepare(&query)?;
-        let mut rows = stmt.query(params![reqid])?;
+        let mut rows = stmt.query(params![message_id])?;
 
         extract_committor_rows(&mut rows)
     }
 
     pub(crate) fn get_commit_status(
         &self,
-        reqid: &str,
+        message_id: u64,
         pubkey: &Pubkey,
     ) -> CommitPersistResult<Option<CommitStatusRow>> {
         let query = format!(
-            "{SELECT_ALL_COMMIT_STATUS_COLUMNS} WHERE reqid = ?1 AND pubkey = ?2"
+            "{SELECT_ALL_COMMIT_STATUS_COLUMNS} WHERE message_id = ?1 AND pubkey = ?2"
         );
         let stmt = &mut self.conn.prepare(&query)?;
-        let mut rows = stmt.query(params![reqid, pubkey.to_string()])?;
+        let mut rows = stmt.query(params![message_id, pubkey.to_string()])?;
 
         extract_committor_rows(&mut rows).map(|mut rows| rows.pop())
     }
 
-    #[cfg(test)]
-    fn remove_commit_statuses_with_reqid(
+    pub fn remove_commit_statuses_with_id(
         &self,
-        reqid: &str,
+        message_id: u64,
     ) -> CommitPersistResult<()> {
-        let query = "DELETE FROM commit_status WHERE reqid = ?1";
+        let query = "DELETE FROM commit_status WHERE message_id = ?1";
         let stmt = &mut self.conn.prepare(query)?;
-        stmt.execute(params![reqid])?;
+        stmt.execute(params![message_id])?;
         Ok(())
     }
 
-    // -----------------
```

Comment:
This is needed as quick lookup to communicate to the user about the status of a bundle to commit.

It cannot just be part of the other table since the row will be cleared once a commit succeeds.

However this table was intended to be kept around for longer such that  a user can query the commit status much later and find all signatures to verify on chain.

Reply:
Not quite sure what this comment refers to